README.md

```diff
@@ -134,13 +135,14 @@ you also need:
 * [Visual Studio 2008](http://www.microsoft.com/visualstudio/en-gb/products/2008-editions)
 * 32 and 64 bit cross compilers
 
-The build will work on any platform, but the tests for IE will be skipped
+The build will work on any platform, but the tests for IE will be 
+skipped
```

Comment:
this is ok @BranTaylor since Markdown doesn't care about line-breaks.  only "double" line breaks.  this will appear the same as if there wasn't one.